apps/dav/src/views/CalDavSettings.vue

```diff
@@ -1,35 +1,30 @@
 <template>
-	<div class="section">
-		<h2>{{ $t('dav', 'Calendar server') }}</h2>
+	<SettingsSection :title="$t('dav', 'Calendar server')"
+		:docUrl="userSyncCalendarsDocUrl">
```

Comment:
Attribute ':docUrl' must be hyphenated     

```suggestion
		:doc-url="userSyncCalendarsDocUrl">
```

Reply:
@CarlSchwan pleas make ``` npm run lint:fix``` and you would not have this problems

Reply:
eslint is always a pain when I have some apps in my repo

```
Oops! Something went wrong! :(

ESLint: 8.7.0

ESLint couldn't find the plugin "eslint-plugin-cypress".

(The package "eslint-plugin-cypress" was not found when loaded as a Node module from the directory "/home/cschwan/nextcloud/workspace/server/apps-extra/activity/cypress".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-cypress@latest --save-dev

The plugin "eslint-plugin-cypress" was referenced from the config file in "apps-extra/activity/cypress/.eslintrc.json".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

Reply:
Found a solution https://github.com/nextcloud/server/pull/32558